### PR TITLE
Mediawiki 1.26 Standard Pass

### PR DESCRIPTION
**Pull request by applehat**
* MediaWiki Coding Standards
* Replaced raw queries with selects
* General Cleanup of Code
* Added extension.json
* Readme Enhancements

### CODE REVIEW

**Comment by applehat** [comment]
@elifoster I will re-review #18 after this pull request is merged in, and fix any conflicts.

I have tested this pull request and it appears to be working, but I'd appreciate it if anyone else would also like to test it out and verify.

File: OreDict.body.php [hunk 1]
```diff
@@ -124,33 +128,93 @@ public function exec($byTag = false){
...
-		if(!isset(self::$mQueries[$fItem][$fMod][$fType])){
-			if($byTag){
+		if (!isset(self::$mQueries[$fItem][$fMod][$fType])) {
+			if ($byTag) {
 				OreDictError::notice("Querying the ore dictionary for Tag = $fItem Mod = $fMod (Call type = $fType)");
-				$query = "SELECT * FROM $fTableName WHERE `tag_name` = $fItem AND ($fMod = '' OR `mod_name` = $fMod) AND ($mfTag & $fType & `flags`) AND ($mfCall & $fType & `flags`) AND ($mfDisp & $fType & `flags`) AND NOT($fDel & `flags`) $sRand $sLim";
+				//$query = "SELECT * FROM $fTableName WHERE `tag_name` = $fItem AND ($fMod = '' OR `mod_name` = $fMod) AND ($mfTag & $fType & `flags`) AND ($mfCall & $fType & `flags`) AND ($mfDisp & $fType & `flags`) AND NOT($fDel & `flags`) $sRand $sLim";
+
+				$result = $dbr->select(
+						$fTableName,
```
Comment (Alexia) [inline OreDict.body.php]: Is this double indented or the HTML being off?

Comment (applehat) [inline OreDict.body.php]: Probably double indented. I'll fix those.

**Comment by Alexia** [comment]
This looks pretty good to me.

**Comment by elifoster** [comment]
Yeah, looks pretty alright yo. I tested all 3 of the special pages and their features briefly.

**Pull request merged**